Tests/Resources/ExpectedTypeScriptCode/Queries/TestQuery7Query.ts

```diff
@@ -142,7 +142,7 @@ export namespace TestQuery7QueryData {
     /**
      * List of the shop's installed fulfillment services.
      */
-    fulfillmentServices: ShopFulfillmentServices[];
+    services: ShopServices[];
```

Comment:
What's the reason for this change?

Reply:
Used `TestQuery7` as a [test for aliases](https://github.com/Shopify/syrup/pull/38/files#diff-fad55641dc2d1348d360c787c13b9bbbfb4742dedd0a7ab373f96f112c36dd82R12).

Reply:
Oh duh.. of course haha. my bad!